scheduler/recovery.go

```diff
@@ -2,21 +2,64 @@ package scheduler
 
 import (
 	"context"
+	"errors"
 	"reflect"
+	"time"
+
+	"github.com/puppetlabs/horsehead/v2/netutil"
+)
+
+const (
+	defaultBackoffMultiplier         = time.Millisecond * 5
+	defaultMaxRetries                = 10
+	defaultResetRetriesTimerDuration = time.Second * 10
 )
 
+// ErrMaxRetriesReached is the error returned by RecoveryDescriptor if the max retries
+// have been reached.
+var ErrMaxRetriesReached = errors.New("RecoveryDescriptor: max retries have been reached")
```

Comment:
We are still using Errawr for all of the errors in this package, although for a v3 it would probably be nice to just go back to regular ol' Go errors.